docs/en/week10/lecture10.sbv

```diff
@@ -4940,118 +4940,118 @@ these features when you perform k-means
 clustering and what you now get is
 
 1:06:52.170,1:06:57.690
-basically for each label for each image
+basically, for each image
 
 1:06:55.260,1:07:00.690
 you know which cluster it belongs to and
 
 1:06:57.690,1:07:02.520
-that becomes its label so in the second
+that becomes its label. So in the second
 
 1:07:00.690,1:07:06.210
-fit step what you do is you train a
+Fit step, what you do is you train a
 
 1:07:02.520,1:07:08.970
-network from scratch so like from random
+network from scratch, so from random
 
 1:07:06.210,1:07:12.660
-bits and you train this network to
+bits, and you train this network to
 
 1:07:08.970,1:07:13.800
-predict just these pseudo labels so if
+predict just these pseudo labels. So they're
 
 1:07:12.660,1:07:15.480
 pseudo because they were basically
 
 1:07:13.800,1:07:17.820
-obtained using clustering so they're not
+obtained using clustering, so they're not
 
 1:07:15.480,1:07:19.290
-really hard labels which were given by
+really hard labels which were given by,
 
 1:07:17.820,1:07:22.230
-say a human annotator
+say, a human annotator.
 
 1:07:19.290,1:07:23.550
-and so now this second network is just
+And so now this second network is just
 
 1:07:22.230,1:07:26.220
 trying to predict these cluster
 
 1:07:23.550,1:07:27.960
-assignments so it takes our image and it
+assignments. So it takes our image and it
 
 1:07:26.220,1:07:29.730
-tries to predict which one of the K
+tries to predict which one of the 
 
 1:07:27.960,1:07:34.070
-clusters that you got from your k-means
+k-clusters that you got from your k-means
 
 1:07:29.730,1:07:36.450
-does this image belong to so a standard
+does this image belong to. So a standard
 
 1:07:34.070,1:07:38.160
-pre train and transfer task is to
+pre-trained and transfer task is to
 
 1:07:36.450,1:07:40.320
-basically perform your pre training so
+basically perform your pre-training, so
 
 1:07:38.160,1:07:42.000
-that's the top row is to perform your
+that's the top row, is to perform your
 
 1:07:40.320,1:07:44.700
-pre training on an objective like
+pre-training on an objective like
 
 1:07:42.000,1:07:46.830
 predicting hashtags or predicting GPS
 
 1:07:44.700,1:07:50.070
-locations and then to evaluate this
+locations, and then to evaluate this
 
 1:07:46.830,1:07:54.150
-feature based by learning linear probe
+feature by learning, say, linear prop (?).
```

Comment:
To me it sounds like "probe". I'm asking Ishan himself.

Reply:
Great, thank you!